public/css/style.css

```diff
@@ -44,6 +44,23 @@ body * {
     padding:10px;
 }
 
+#message a {
+    font-size:0.9em;
+    text-align:center;
+    -moz-border-radius:2px;
```

Comment:
No need for prefixed versions, since border-radius is [widely supported](http://caniuse.com/#feat=border-radius).